dotcom-rendering/src/frontend/feFootballMatchListPage.ts

```diff
@@ -1,24 +1,17 @@
-import type { FEFrontConfig } from './frontend/feFront';
-import type { EditionId } from './lib/edition';
-import type { FooterType } from './types/footer';
-import type { FENavType } from './types/frontend';
+import type { FEFootballDataPage, FERound } from './feFootballDataPage';
 
-type FECompetitionSummary = {
+export type FECompetitionSummary = {
```

Comment:
Does this belong in `feFootballDataPage` since the model is shared with tables?

Reply:
Good spot, moved it to feFootballDataPage